includes/wikia/parser/templatetypes/handlers/NavboxTemplate.class.php

```diff
@@ -28,20 +28,30 @@ private static function process( $html ) {
 		preg_match_all( $markerRegex, $html, $markers );
 
 		foreach ( array_unique( $markers[ 2 ] ) as $marker ) {
-			// matches elements in between start and end marker tags
-			// <marker>(not </marker>)...</marker>
-			$html = preg_replace(
-				'/(<|&lt;)' . $marker . '(>|&gt;)' .
-				'((?!(<|&lt;)\\/' . $marker . '(>|&gt;)).)*' .
-				'(.*)' .
-				'(<|&lt;)\\/' . $marker . '(>|&gt;)/isU',
-				// replacement
-				'<div data-type="navbox">$0</div>',
-				$html
-			);
+			$html = static::replaceMarker( $marker, $html );
+		}
+
+		return $html;
+	}
+
+	private static function replaceMarker( $marker, $html ) {
```

Comment:
can we add an info that this is an experiment and should be cleaned up with ticket DAT-XYZ